docs/index.rst

```diff
@@ -9,8 +9,11 @@ This is the documentation for sunkit-magex.
    :caption: Contents:
 
    installing
+   heritage
    generated/gallery/index
-   reference/index
+   reference/index   
```

Comment:
These whitespace issues, can be fixed by either running pre-commit, using the pre-commit bot or `tox -e codestyle`.